core/iwasm/libraries/wasi-nn/cmake/Findtensorflow_lite.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+find_library(TENSORFLOW_LITE 
+     NAMES tensorflow-lite
+     HINTS ${CMAKE_LIBRARY_PATH}
+)
+
+if(NOT EXISTS ${TENSORFLOW_LITE})
+    if (NOT EXISTS "${WAMR_ROOT_DIR}/core/deps/tensorflow-src")
+        execute_process(COMMAND ${WAMR_ROOT_DIR}/core/deps/install_tensorflow.sh
+                        RESULT_VARIABLE TENSORFLOW_RESULT
+        )
+    else ()
+        message("Tensorflow is already downloaded.")
+    endif()
+    set(TENSORFLOW_SOURCE_DIR "${WAMR_ROOT_DIR}/core/deps/tensorflow-src")
+
+    if (WASI_NN_ENABLE_GPU EQUAL 1)
+    # Tensorflow specific:
+    # * https://www.tensorflow.org/lite/guide/build_cmake#available_options_to_build_tensorflow_lite
+    set (TFLITE_ENABLE_GPU ON)
+    endif ()
+
+    include_directories (${CMAKE_CURRENT_BINARY_DIR}/flatbuffers/include)
+    include_directories (${TENSORFLOW_SOURCE_DIR})
+    add_subdirectory(
+        "${TENSORFLOW_SOURCE_DIR}/tensorflow/lite"
+        "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL) 
+
+else()
+    find_path(TENSORFLOW_LITE_INCLUDE_DIR
+    NAMES tensorflow/lite/interpreter.h
+    HINTS ${CMAKE_LIBRARY_PATH}../include
```

Comment:
`CMAKE_LIBRARY_PATH` is empty, is it expected?

Reply:
The user will set it if they have installed libraries already within the target. 

This is coming test code for it
https://github.com/midokura/wasm-micro-runtime/blob/c9997578f1a39662900cddf5c8a7b84225415939/core/iwasm/libraries/wasi-nn/test/Dockerfile.vx-delegate#L88

Reply:
I remove it because if we use CMAKE_LIBRARY_PATH, it can be omitted from find_path, find_package